components/cost-management/staging/policies/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,512 @@
+    is denied if the `cost-center` label is invalid.
+  steps:
+    - name: Apply RBAC
+      try:
+        - apply:
+            file: ../kyverno_rbac.yaml
+    - name: Apply Kyverno Cluster Policy and assert it exists
+      try:
+        - apply:
+            file: ../validate_namespace_labels.yaml
+        - assert:
+            file: chainsaw-assert-clusterpolicy.yaml
+            template: true
+            bindings:
+              - name: cluster_policy_name
+                value: validate-cost-center-label
+    - name: Attempt to create a tenant namespace with invalid cost-center label
+      try:
+        - create:
+            file: ./resources/namespace-cost-center.yaml
+            template: true
+            bindings:
+              - name: namespace
+                value: tenant
+              - name: cost_center
+                value: "invalid-should-be-all-numbers"
+            expect:
+            - match:
+                kind: Namespace
+              check:
+                ($error != null): true
+
+---
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: pod-creation-with-kyverno-unavailable
+spec:
+  concurrent: false
+  description: |
+    Simulates a scenario where the Kyverno webhook is unavailable by scaling down the admission controller.
+    Verifies that pods can still be created successfully without the cost-center or insights labels being propagated,
+    as the policy's failurePolicy is set to Ignore.
+  steps:
+    - name: Create a tenant namespace with cost-center label
+      try:
+        - create:
+            file: ./resources/namespace-cost-center.yaml
+            template: true
+            bindings:
+              - name: namespace
+                value: tenant-with-cost-center
+              - name: cost_center
+                value: "123"
+    - name: Apply RBAC
+      try:
+        - apply:
+            file: ../kyverno_rbac.yaml
+    - name: Apply Kyverno Cluster Policy and assert it exists
+      try:
+        - apply:
+            file: ../propogate_namespace_labels.yaml
+        - assert:
+            file: chainsaw-assert-clusterpolicy.yaml
+            template: true
+            bindings:
+              - name: cluster_policy_name
+                value: propagate-cost-management-labels
+    - name: Scale down Kyverno admission controller
+      try:
+        - apply:
+            resource:
+              apiVersion: apps/v1
+              kind: Deployment
+              metadata:
+                name: kyverno-admission-controller
+                namespace: konflux-kyverno
+              spec:
+                replicas: 0
+
+    - name: Wait for Kyverno admission controller to be down
+      try:
+        - script:
+            content: |
+              echo "Waiting for Kyverno admission controller to scale down..."
+              sleep 15
+              kubectl wait --for=delete pod -l app.kubernetes.io/name=kyverno -n konflux-kyverno --timeout=60s || true
```

Comment:
maybe we can use the [wait helper](https://kyverno.github.io/chainsaw/0.2.3/operations/helpers/wait/) here.

Also
* the `|| true` at the end is very strange to me
* is the `sleep 15` needed?